src/BloomExe/CollectionTab/LibraryView.cs

```diff
@@ -66,6 +66,11 @@ public partial class LibraryView :  UserControl, IBloomTabArea
 			// Set the background image for Mono because the background color does not paint,
 			// and if we override the background paint handler, the default styling of the child 
 			// controls is changed.
+
+			// We are getting an exception if none of the buttons are visible. The tabstrip is set
+			// to Dock.Top which results in the height being zero if no buttons are visible.
+			if ((_toolStrip.Height == 0) || (_toolStrip.Width == 0)) return;
```

Comment:
Currently it cannot resize after the buttons are hidden because they are made visible/invisible in the constructor only, before the BackgroundColorsForLinux() function is called.